packages/layout/src/image/ninePatch.js

```diff
@@ -0,0 +1,366 @@
+// import { createCanvas, loadImage } from "canvas";
+import { Canvas, loadImage, Image } from "skia-canvas";
+
+class NinePatch {
```

Comment:
Can you explain what is this?

Reply:
@diegomura  I have created background image with ninepatch that automatically create background canvas with shadow for background image we have to just pass image for that and it look lile below.
![image](https://github.com/user-attachments/assets/f16f377f-57c6-47c3-a715-146c36bd9c27)
![image](https://github.com/user-attachments/assets/89ddd499-506a-456b-a064-9dea08251816)
![image](https://github.com/user-attachments/assets/9a5b7856-9b6a-4753-a4ca-1845ba4565e5)

Reply:
Yes that is box shadow but we have create image look like box shadow and provide that image as background so that's why it's related to background image.